django/contrib/auth/admin.py

```diff
@@ -80,9 +82,11 @@ def get_form(self, request, obj=None, **kwargs):
         return super(UserAdmin, self).get_form(request, obj, **defaults)
 
     def get_urls(self):
-        return [
-            url(r'^(.+)/password/$', self.admin_site.admin_view(self.user_change_password), name='auth_user_password_change'),
-        ] + super(UserAdmin, self).get_urls()
+        return [url(
```

Comment:
@timgraham noted!

About the line... I just changed it because it was *really* long, above 119 characters (my _hard_ limit).

I'll revert that change and update replace `MovelClass` with `ModelAdmins` (sorry for this).

Thank you!